packages/frontend/src/pages/scaling-activity/view/ActivityView.tsx

```diff
@@ -38,7 +44,14 @@ export function ActivityView({ items }: ActivityViewProps) {
       name: '#',
       alignRight: true,
       minimalWidth: true,
-      getValue: (_, index) => index + 1,
+      getValue: (entry, index) => {
+        if (entry.isVerified === false) {
+          return (
+            <UnverifiedWarning message="This project includes unverified contracts" />
```

Comment:
Is there any place in the code which uses a different message? If not, let's make `"This project includes unverified contracts"` part of `UnverifiedWarning.tsx` component in order to remove possible discrepancies in this message across the codebase.